.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
Curious (only curious 😉), why the choice of `.js` vs. pure JSON?

Reply:
Ha! This same debate was just had in the opposite direction ... some argument about keeping everything in the same language (JS) was made. I'm going to stay out of this one.

Reply:
I do prefer something that allows comments which means yaml or js style.  But otherwise, I don't think it matters in simple declarative use cases like this.